css/main2.css

```diff
@@ -96,6 +96,12 @@ H2.titletagline {
     font-size: 10pt;
 }
 
+H2.column {
+    text-align: left;
+    font-weight: bold;
+    font-size: 12pt;
```

Comment:
This is increasing the size of the `Find us on social media!` and `Donate!` text disproportionately to the rest of the text when using Chrome. I do see that Firefox and Edge seem to not do that. Perhaps find a way to make it consistent across browsers?

Reply:
This appears to be something related to how my laptop accesses my internal web server. Using `localhost` in Chrome behaves differently than when I use an explicit IP address. I'll have to investigate this on my end, but it's not your problem.